ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -310,6 +313,12 @@ TorchCore::validateOutputTensor (const at::Tensor output, unsigned int idx)
   at::Tensor sliced_output = output.slice (0);
   c10::IntArrayRef sliced_output_sizes = sliced_output.sizes ();
 
+  /** if idx is in bounds */
+  if (outputTensorMeta.num_tensors <= idx) {
+    ml_loge ("Invalid output meta: different number of outputs");
```

Comment:
Please be more specific for error messages. For example, you need to show the desired number of outputs and the actual number of outputs and describe how the user should update their code.

Reply:
Updated the error message to be more specific and describe to user as to how resolve this error.